### PR TITLE
Add C14 formula

### PR DESCRIPTION
**Pull request by edouardb**
Add Online [C14](https://www.online.net/en/c14) [CLI](github.com/online-net/c14-cli) formula

### CODE REVIEW

**Comment by Solvik** [comment]
👍

File: Formula/c14.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class C14 < Formula
```
Comment (fxcoudert) [inline Formula/c14.rb]: Given the project name, the formula should be `c14-cli`

File: Formula/c14.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  url "https://github.com/online-net/c14-cli/archive/0.1.tar.gz"
+  sha256 "cff3597273daff87e8d6e85cfef2b4d83400f0a0a905f39a4a67560b4966513c"
+  head "https://github.com/online-net/c14-cli.git"
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "80a12f0378bf0cc4d6821b9dd9c3381b89771e95f523d57d876b7d7e316affa1" => :sierra
+    sha256 "792be6c0a7b700ad9ae6c618dde4e6e9fbf7a35e1c204ebf4d366c805d191cec" => :el_capitan
+    sha256 "d5d5189fd3a4e2fcf368717e770a811f5a7c4558022b450fa330503b106d1c7e" => :yosemite
+  end
```
Comment (fxcoudert) [inline Formula/c14.rb]: Remove the `bottle` block, it will be added when merging.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/c14-cli.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  head "https://github.com/online-net/c14-cli.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = buildpath
+    (buildpath/"src/github.com/online-net/c14-cli").install Dir["*"]
+
+    system "go", "build", "-o", "#{bin}/c14", "-v", "-ldflags", "-X  github.com/online-net/c14-cli/pkg/version.GITCOMMIT=homebrew", "github.com/online-net/c14-cli/cmd/c14/"
```
Comment (DomT4) [inline Formula/c14-cli.rb]: [This kind of style](https://github.com/Homebrew/homebrew-core/blob/0c14c6c7685c44e8771b1d3f2732cccb66dcfd75/Formula/wellington.rb#L37-L39) is a nicer format to read than one extremely long line FWIW.

**Review by DomT4 — APPROVED** [review]

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @edouardb for your contribution to Homebrew!

**Pull request closed without merging**